src/app/tabs/tabs.page.scss

```diff
@@ -8,4 +8,19 @@
 
 .margin-extra-left {
     margin-left: 7%;
-}
\ No newline at end of file
+}
+
+
+// :host {
+//     --background: var(--ion-item-background, var(--ion-background-color, #fff));
```

Comment:
Remove code instead of commenting out

Reply:
Its is not a common header, It is a local header of "My Day" page